src/locale/locales/id/messages.po

```diff
@@ -0,0 +1,2478 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-12-28 11:56+07000\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: id\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: danninov\n"
+"Language-Team: danninov\n"
```

Comment:
Thanks for all your hard work @danninov @Thinkbyte1024 Please add yourself to the Language Team here

Reply:
Just a minor suggestion: Changing my username to all lowercase. :-)

`thinkbyte1024`